src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -8,6 +8,9 @@ import { connect } from 'react-redux';
 import { toggleLoginModal as toggleLoginModalAction } from 'platform/site-wide/user-nav/actions';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import Telephone, {
+  CONTACTS,
+} from '@department-of-veterans-affairs/component-library/Telephone';
```

Comment:
I'm recommending the web component, not the React component. The `CONTACTS` import can still be useful though.

```suggestion
import  { CONTACTS } from '@department-of-veterans-affairs/component-library';
```